Message validation and flashMessage function

kawa18sima opened the pull request:
# WHAT
Message Validation
Display function of flash message at message transmission
# WHY
To make it visually easy for the user

Commit:
Define validation of message body column

Add presence: true to the body column of message

Commit:
Added flash message when sending message

Addition of flash message at the time of message transmission success
and message transmission failure

kawa18sima commented:
以下フラッシュメッセージを表示させたときのviewです。
<img width="1440" alt="2017-05-10 23 25 51" src="https://cloud.githubusercontent.com/assets/24731933/25904602/b1db2dce-35da-11e7-94d8-14829aca1849.png">
<img width="1440" alt="2017-05-10 23 37 55" src="https://cloud.githubusercontent.com/assets/24731933/25904604/b3e66a0c-35da-11e7-984f-7a3ec419712a.png">

kinshotomoya commented on app/controllers/messages_controller.rb:
```diff
@@ -10,9 +10,10 @@ def index
     if message.save
+      redirect_to group_messages_path(params[:group_id]), notice:  "メッセージ送信成功"
     else
+      redirect_to group_messages_path(params[:group_id]), alert: "メッセージを入力してください"
```
renderにしよう！！
redirectとrenderの違いを一回調べてみて！

kinshotomoya reviewed (commented):
コメントしました！

Commit:
Edit Message Controller

Fix render code when message transmission fails

In this case, since the index view is called in the create action, it
also fixes how to write the variable

kawa18sima commented:
修正しました！
一つ確かめたいことがあります。
renderにした場合viewを置き換えるみたいな感じになるので、createアクションでindexのviewが呼ばれるという解釈であってますか？

kinshotomoya reviewed (commented):
みました！
いい感じです！！！！！！

renderはviewを同じcontroller内でviewを変化させる！
その解釈であってるね！

まーじしましょ！！！！！！！！！！！！！！！！！！！！
